web/packages/dmt-core/src/components/UploadFileButton.tsx

```diff
@@ -48,6 +48,7 @@ export function UploadFileButton(props: {
   const textInput = useRef<HTMLInputElement>(null)
   const [error, setError] = useState<string | undefined>()
   const [loading, setLoading] = useState<boolean>(false)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore